src/kbmod/file_utils.py

```diff
@@ -183,6 +196,12 @@ def load_results_file(filename):
             usecols=(1, 3, 5, 7, 9, 11, 13),
             names=["lh", "flux", "x", "y", "vx", "vy", "num_obs"],
         )
+
+        # If there is only one line, numpy will return a 1-d array.
+        # Make it 2-d for consistency.
+        if len(results.shape) == 0:
```

Comment:
Could you use the `ndmin` parameter when calling `genfromtxt`? That can force the `results` object to have at least 2 dimensions.

Reply:
Good call. That makes the function a lot cleaner.